manifests/jenkins.yaml

```diff
@@ -136,8 +136,11 @@ objects:
             name: ${JENKINS_SERVICE_NAME}-data
           # DELTA: mount c-as-c config map
           - name: ${JENKINS_SERVICE_NAME}-casc-cfg
-            mountPath: /var/lib/jenkins/configuration-as-code
+            mountPath: /var/lib/jenkins/configuration-as-code/base
```

Comment:
is this a harmless change? we're changing the mountpath of something, but I don't see another corresponding change somewhere else to adjust for it (in this same commit).

Reply:
Yeah, it should be harmless. JCasC recurses into directories. So now it'll look in both the `base` and the `splunk` subdirs.